Makefile

```diff
@@ -110,7 +110,7 @@ bin/pfcmd: src/pfcmd.c
 	$(CC) -O2 -g -std=c99  -Wall $< -o $@
 
 bin/ntlm_auth_wrapper: src/ntlm_auth_wrap.c
-	$(CC) -g -std=c99 -Wall $< -o $@
+	$(CC) -g -std=c99 -lcurl -lcjson -Wall $< -o $@
```

Comment:
As mentioned before, add the packages in the BuildRequires:

https://github.com/inverse-inc/packetfence/blob/devel/rpm/packetfence.spec#L30

Do the same thing for Debian.